Three/include/CGAL/Three/Viewer_interface.h

```diff
@@ -41,6 +43,14 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
   Q_OBJECT
 
 public:
+  enum OpenGL_program_IDs { PROGRAM_WITH_LIGHT,
+                            PROGRAM_WITHOUT_LIGHT,
+                            PROGRAM_WITH_TEXTURE,
+                            PROGRAM_WITH_TEXTURED_EDGES,
+                            PROGRAM_INSTANCED,
+                            PROGRAM_INSTANCED_WIRE,
+                            NB_OF_PROGRAMS };
+
   Viewer_interface(QWidget* parent) : QGLViewer(CGAL::Qt::createOpenGLContext(), parent) {}
```

Comment:
Ideally, that enum should not be defined in both `Scene_item` and `Viewer_interface`. I do not know how to do a `using` of an enum from another class.